webrender/src/frame_builder.rs

```diff
@@ -1551,8 +1547,11 @@ impl FrameBuilder {
         &self,
         clip_and_scroll: &ClipAndScrollInfo
     ) -> Option<PackedLayerIndex> {
-        let group_index = self.clip_scroll_group_indices.get(&clip_and_scroll).unwrap();
-        let clip_scroll_group = &self.clip_scroll_group_store[group_index.0];
+        let group_id = self.clip_scroll_group_indices[&clip_and_scroll];
+        //Note: this ignores `group_ref.visibility` intentionally
```

Comment:
This comment doesn't seem right with this version of the patch?

Reply:
good eye! fixed now